max_x: 30.0 # [m]
max_y: 50.0 # [m]
max_z: 10.0 # [m]
